MdeModulePkg/Bus/Pci/PciHostBridgeDxe/PciHostResource.h

```diff
@@ -24,6 +24,15 @@ typedef enum {
   TypeMax
 } PCI_RESOURCE_TYPE;
 
+// MU_CHANGE begin
+// Only processor type X64 has IO resource
+#if defined (MDE_CPU_X64)

```

Comment:
Was IA32 intentionally excluded?

```suggestion
#if defined (MDE_CPU_IA32) || defined (MDE_CPU_X64)

```

Reply:
Hi Michael. This makes sense. Thanks for the suggestion!